src/docfx/build/toc/parser/TableOfContentsInputItem.cs

```diff
@@ -4,11 +4,13 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Web;
+using Newtonsoft.Json;
 
 namespace Microsoft.Docs.Build
 {
     internal class TableOfContentsInputItem
     {
+        [JsonProperty(Required = Required.DisallowNull)]
         public string Name { get; set; }
```

Comment:
>Name [](start = 22, length = 4)

add a test?

Reply:
The related test was the one you suggest to be skipped

---
In reply to: [201892945](https://github.com/dotnet/docfx/pull/3052#discussion_r201892945) [](ancestors = 201892945)

Reply:
ho I see, can it be a spec test?

---
In reply to: [201893698](https://github.com/dotnet/docfx/pull/3052#discussion_r201893698) [](ancestors = 201893698,201892945)

Reply:
do you mean a e2e test? but how could we skip a e2e test within .yml file?

---
In reply to: [201895244](https://github.com/dotnet/docfx/pull/3052#discussion_r201895244) [](ancestors = 201895244,201893698,201892945)